### ISSUE TITLE
Terminal crashes with error 0x8007007A on launch

### ISSUE THREAD

**Issue by Servellia** [microsoft/terminal#7551]
I'm running a German localization on this machine, so translations might be a bit off.

Expected result:
Windows Terminal is launching

Actual result:
Windows terminal is crashing on launch.

I've installed Windows Terminal from the Store and receive the following error upon launching it:

![terminal-error1](https://user-images.githubusercontent.com/2832279/92359656-ea78fd80-f0eb-11ea-8448-6555d9570f16.JPG)

I do believe the correct translation to this is:

_data area passed to a system call is too small_

The Event Viewer Shows the following entry:

`0x8007007A: Der Prozess für das Paket Microsoft.WindowsTerminal_1.2.2381.0_x64__8wekyb3d8bbwe kann nicht erstellt werden, da beim Konfigurieren der Laufzeit ein Fehler aufgetreten ist. [LaunchProcess]`

**Comment by DHowett** [comment]
What version of Windows are you using? Have you reconfigured the "Application Install path" (in the Storage > "Change where new content is saved" section of Settings)

![image](https://user-images.githubusercontent.com/189190/92507723-40b57180-f1bc-11ea-944d-51eca3fe7a62.png)

**Comment by Servellia** [comment]
The machine is running on 1909 

No, I have not changed the default location, as it is running on a single partition and the only drive available is the default C drive.

Best regards

**Comment by DHowett** [comment]
This reminded me of #2914.

Does this information help? https://support.microsoft.com/en-us/help/4073393/fix-the-data-area-passed-to-a-system-call-is-too-small-error-when-you

Do you have SQL Server installed? :smile:

**Comment by DHowett** [comment]
I'm sorry, I meant #2912.

**Comment by Servellia** [comment]
I've already read that, but ignored the "desktop bridge" part of the conversation. I'll update SQL Server from 14.0.100.169 to 14.0.3335.7 and reply back.

For what it's worth, I found a workaround yesterday evening, by gettingaccess to `C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_1.2.2381.0_x64__8wekyb3d8bbwe` and starting wt.exe directly from there. This is circumventing the issue and the terminal comes up successfully. So I have pinned this directly to the taskbar instead of calling it via the start menu.
